### PR TITLE
Standardize error logging in express app

### PR DESCRIPTION
**Pull request by psoleckimoj**
- Log out errors using an object to record more info
- Fix test with broken async await test and code

### CODE REVIEW

**Review by davidthomas22 — DISMISSED** [review]
LGTM

File: server/repositories/hubFeaturedContent.js [hunk 1]
```diff
@@ -9,11 +9,15 @@ function hubFeaturedContentRepository(httpClient) {
...
+    try {
+      const response = await httpClient.get(endpoint, { query });
 
-    if (!Array.isArray(response)) return [];
+      if (!Array.isArray(response)) return [];
 
-    return R.map(featuredContentTileResponseFrom, response);
+      return R.map(featuredContentTileResponseFrom, response);
+    } catch (e) {
+      return [];
```
Comment (LucasCairns) [inline server/repositories/hubFeaturedContent.js:19]: Do we want to log the exception?

Comment (psoleckimoj) [reply server/repositories/hubFeaturedContent.js]: Removed this now

**Review by LucasCairns — APPROVED** [review]
LGTM

**Pull request merged**